blockchain/tx_pool_test.go

```diff
@@ -65,6 +68,15 @@ func init() {
 	eip1559Config.LondonCompatibleBlock = common.Big0
 	eip1559Config.EthTxTypeCompatibleBlock = common.Big0
 	fork.SetHardForkBlockNumberConfig(eip1559Config)
+
+	cpy = *params.TestChainConfig
+	kip71Config = &cpy
+	kip71Config.KIP71CompatibleBlock = common.Big0
+	kip71Config.IstanbulCompatibleBlock = common.Big0
+	kip71Config.LondonCompatibleBlock = common.Big0
+	kip71Config.EthTxTypeCompatibleBlock = common.Big0
+	kip71Config.Governance = &params.GovernanceConfig{KIP71: params.GetDefaultKip71Config()}
+	fork.SetHardForkBlockNumberConfig(eip1559Config)
```

Comment:
```suggestion
	fork.SetHardForkBlockNumberConfig(kip71Config)
```

Reply:
I think we need to delete it. kip71 doesn't use `fork` instance. Thanks!